carbon/app.py

```diff
@@ -16,83 +16,148 @@
 from carbon.db import persons, orcids, dlcs, engine, aa_articles
 
 AREAS = (
-    'ARCHITECTURE & PLANNING AREA', 'ENGINEERING AREA',
-    'HUMANITIES, ARTS, & SOCIAL SCIENCES AREA', 'SCIENCE AREA',
-    'SLOAN SCHOOL OF MANAGEMENT AREA', 'VP RESEARCH', 'CHANCELLOR\'S AREA',
-    'OFFICE OF PROVOST AREA', 'PROVOST AREA',
+    "ARCHITECTURE & PLANNING AREA",
```

Comment:
We should likely move these constants to the `config` module that will be created soon

Reply:
I agree!